tools/sof-dump-status.py

```diff
@@ -435,7 +435,7 @@ def dump_dapm(dapm, filter = "all"):
             print(run_status['status'])
         exit(0)
 
-    if ret_args['export'] is True:
+    if ret_args['export'] is not None:
```

Comment:
same here, move this two patches earlier when this was added first.

Reply:
thsi patch needs to be squashed with the previous one